narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -25,24 +30,46 @@ function check_if_tag_exists {
 	fi
 }
 
-function check_ignore_zip_file {
-	local hotfix_zip_file="${1}"
-	local release_version="${2}"
+function check_ignore_via_argument {
+	local ignore_zip_file="${1}"
 
-	local file_url="${zip_directory_url}/${hotfix_zip_file}"
+	if [[ "x${ignore_zip_file}" =~ x*${hotfix_zip_file}* ]]
+	then
+		lc_log DEBUG "Ignoring '${file_url}'."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_OK}"
+	else
+		lc_log DEBUG "The file is not ignored via command line argument."
```

Comment:
From this message it is more obvious that we should rename ignore to exclude.

Reply:
BTW I would rather write "The file ${hotfix_zip_file} is not on the command line defined ignore list" or something like that.